quill-jasync-zio-postgres/src/main/scala/io/getquill/context/qzio/PostgresJAsyncContextConfig.scala

```diff
@@ -1,4 +1,4 @@
-package io.getquill.context.zio
+package io.getquill.context.qzio
```

Comment:
I made this change because the fact that it was named `zio` was clashing with all the ZIO imports, the compiler complaining for example that `Task` was not defined in `io.getquill.context.zio`